logging/import-logs/main.py

```diff
@@ -79,6 +79,18 @@ def _day(blob_name: str) -> int:
     return int(blob_name[offset : offset + 2])
 
 
+def _is_valid_import_range() -> bool:
+    if START_DATE > END_DATE:
+        eprint("Start date of the import time range should be earlier than end date")
+        return False
+    # avoid importing time range older than 29 days
+    today = date.today()
+    if (today - START_DATE).days > 29 or (today - END_DATE).days > 29:
```

Comment:
nit: the second part of this statement (`(today - END_DATE).days > 29`) will never be true, because START_DATE > END_DATE, so it could be skipped.

Reply:
nit: I agree but it does not influence a performance and the statement is more verbose this way.

Reply:
SG. Thanks for taking a look.